src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VelgPeriode.tsx

```diff
@@ -64,14 +53,16 @@ const VelgPeriode: React.FC<IProps> = ({
     const erLesevisning = vurderErLesevisning();
 
     return (
-        <MarginSkjemaGruppe
-            feilmeldingId={vilkårPeriodeFeilmeldingId(redigerbartVilkår.verdi)}
-            feil={
+        <MarginFieldset
+            errorId={vilkårPeriodeFeilmeldingId(redigerbartVilkår.verdi)}
+            error={
                 redigerbartVilkår.verdi.periode.valideringsstatus === Valideringsstatus.FEIL &&
                 visFeilmeldinger
                     ? redigerbartVilkår.verdi.periode.feilmelding
                     : ''
             }
+            legend="Periode for vurderingen"
+            hideLegend
```

Comment:
Legend er obligatorisk i Fieldset. Setter den skjult så den kun er tilgjengelig for skjermleser